CONTRIBUTING.md

```diff
@@ -2,31 +2,109 @@
 
 First, thanks for taking the time to contribute to the project! Your support is highly appreciated - kudos!
 
-The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules.
+Use your best judgment and feel free to propose changes to this document in a pull request.
 
 ## How to contribute
 
-Before kicking the ball, please **do NOT report security vulnerabilities in public issues**! Instead, report with the CloudFoundry Foundation team first <security@cloudfoundry.org> and give us some time to fix it in a fashion time matter before disclosing it to the public. For more information check the CloudFoundry Security [page](https://www.cloudfoundry.org/security/).
+Please report to the CloudFoundry foundation <security@cloudfoundry.org> and we will fix the issues as quickly as we can before the the details are shared with the public
```

Comment:
I think it misses the context of "Security issues". Otherwise it can be misunderstood

Reply:
Security vulnerabilities should be enough.

Reply:
wait.. wasn't that removed?